koiki/create_delivery.py

```diff
@@ -10,9 +8,8 @@ class CreateDelivery():
 
     def __init__(self, order):
         self.order = order
-        self.by_vendor = self._by_vendor(order['line_items'])
-        self.shipping = Shipping(order['shipping'])
-        self.billing = Billing(order['billing'])
+        self.shipping = Shipping(order.data['shipping'])
```

Comment:
IMO `.data` misses the point of encapsulating the data structure in a class. If we need to, we should implement the `billing` and `shipping` methods and not return the data structure we were aiming to wrap.